### DISCUSSION TITLE
Testing like Jest in Rome Tool

### DISCUSSION THREAD

**Discussion by nhattran998** [rome/tools#1552] [Q&A] [answered]
I see the docs say Rome design to replace some tools, like Jest. But I don't see where to configure and use that in the document. 
If it not already exist, hopefully, your team add the testing section in Roadmap for everybody can track them!
Thanks 🤗

**Comment by ematipico** [comment, marked as answer]
Yes, the intention is also to open up a testing framework. Internally we already have a testing framework that we use. You can say that Rome tests itself. Although, the testing framework is not ready test to be released because it needs a working bundler. 

Once we have a working bundler a mock system that works, that would be ready to be released.

**Reply by nhattran998** [reply]
So thankful for that 😁 . I think your team should add some docs to upgrade the progress in the `https://rome.tools/` website to anyone who know that it's working on progress 😄.

**Reply by ematipico** [reply]
About that, we recently discussed that on Discord and we are going to use a combination of milestones and "Projects" to coordinate our works and show what is being worked. 

I am not sure about the website, we could consider it! Thank you for your suggestion.
